src/main/java/com/iridium/iridiumskyblock/api/IridiumSkyblockAPI.java

```diff
@@ -263,4 +263,15 @@ public World getEndWorld() {
         return iridiumSkyblock.getIslandManager().getEndWorld();
     }
 
+    /**
+     * Returns whether or not the specified world is from IridiumSkyblock.
+     *
+     * @param world Thw world that should be checked
+     * @return true if it is a world used by IridiumSkyblock
+     * @since 3.0.7
+     */
+    public boolean isIslandWorld(World world) {
+        return getWorld() == world || getNetherWorld() == world || getEndWorld() == world;
```

Comment:
I think we should have this method in IslandManager and link this method to the IslandManager's method since thats what all the other stuff does

Reply:
It's more of a utility method for checking if IridiumSkyblockAPI#getWorld(), IridiumSkyblockAPI#getNetherWorld() or IridiumSkyblockAPI#getEndWorld() equal to another world. It makes more sense to be here in my opinion